__snapshots__/index.spec.js.snap

```diff
@@ -2460,7 +2460,7 @@ _   underscore
 exports[`markdown-to-jsx compiler headings adds an "id" attribute to headings for deeplinking purposes 1`] = `
 
 <h1 data-reactroot
-    id="this-is-a-very-complicated-header"
+    id="this-is-a-very-complicated%3e-header"
```

Comment:
So I'd prefer if only non-latin characters are encoded. `>` is definitely not a character I'd want to show up in the ID.